src/runner.go

```diff
@@ -125,7 +124,12 @@ func processSignedScript(incomingContent []byte) string {
 	// Execute script
 	log.Infoln("Executing script...")
 	cmd := exec.Command(yamlContent.Vars.Interpreter, scriptFileName) //nolint:gosec
-	setEnvVariablesForCommand(cmd, yamlContent.Vars.ContentVars)
+	cmd.Env = os.Environ()
+
+	variables := yamlContent.Vars.ContentVars
+	variables["LOG_LEVEL"] = *config.ScriptLogLevel
+	setEnvVariablesFromYaml(cmd, variables)
+	log.Infoln("Script is being executed with following env variables: ", cmd.Env)
```

Comment:
Don't know if I should leave this, I initially added it to see what rhc passes down to us, but now I am realizing that we may not want to print it to logs, I don't expect any passwords but we never know

Reply:
removed!